lib/decidim/file_authorization_handler/version.rb

```diff
@@ -2,11 +2,11 @@
 
 module Decidim
   module FileAuthorizationHandler
-    DECIDIM_VERSION = "0.26.2"
+    DECIDIM_VERSION = "0.27.1"
 
     # Uses the latest matching Decidim version for
     # - major, minor and patch
     # - the optional extra number is related to this module's patches
-    VERSION = "#{DECIDIM_VERSION}.5"
+    VERSION = "#{DECIDIM_VERSION}.5".freeze
```

Comment:
Maybe we must increment this version, what do you think?

Reply:
Done. It does not increment as version really changes from `0.26.2.5` to `0.27.1.0`